src/affine.coffee

```diff
@@ -64,8 +64,8 @@ class affine2d
       " #{@m11.toPrecision(3)}]   V = (" +
        "#{@v0.toPrecision(3)}, " +
       " #{@v1.toPrecision(3)})   scale = " +
-      @getXScale().toPrecision(3) + " x " + 
-      @getYScale().toPrecision(3)
+      @getXCenter().toPrecision(3) + " x " +
```

Comment:
I'm a little confused about this change. The printout says "scale = " and then it was previously printing `getXScale()` and `getYScale()`, but after your change it says "scale =" and then prints the center. Can you explain?